File: Workout/app/src/main/java/com/example/workout/DetailActivity.java
package com.example.workout;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class DetailActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_detail);
        WorkoutDetailFragment frag = (WorkoutDetailFragment)getSupportFragmentManager().findFragmentById(R.id.detail_frag);
        frag.setWorkout(1); //액티비티에서 운동id설정
    }
}

File: recycler_view_staggered_grid/app/src/main/java/com/cocokaribou/recycler_view_staggered_grid/DetailActivity.kt
package com.cocokaribou.recycler_view_staggered_grid

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity
import com.bumptech.glide.Glide
import com.bumptech.glide.load.engine.DiskCacheStrategy
import com.cocokaribou.recycler_view_staggered_grid.databinding.ActivityDetailBinding

class DetailActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Log.e(javaClass.simpleName, "onCreate()")
        val binding = ActivityDetailBinding.inflate(layoutInflater)
        setContentView(binding.root)
        intent.getStringExtra("photo")?.let{ url ->
            Glide.with(binding.ivPhotoImgDetail.context)
                .load(url)
                .into(binding.ivPhotoImgDetail)
        }
    }

}